packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/editModal.gql.js

```diff
@@ -4,7 +4,7 @@ export const GET_SELECTED_PAYMENT_METHOD = gql`
     query getSelectedPaymentMethod($cartId: String!) {
         cart(cart_id: $cartId) @connection(key: "Cart") {
             id
-            selectedPaymentMethod: selected_payment_method {
```

Comment:
I removed the alias to avoid potential misses of cache info. I wasn't sure if aliases are used when caching data but Apollo docs specify that aliases are intended for collisions such as having two `cart` queries for different `ids` within a single template:

```js
query getCarts {
  firstCart: cart(id: 1) { ... }
  secondCart: cart(id: 2) { ... }
}
```